app/controllers/users_controller.rb

```diff
@@ -61,31 +59,35 @@ def invite_resource(&block)
       u.skip_invitation = true
     end
 
-    user = User.find_by_email(resource_params[:email])
-
-    if current_user.admin?
-      if params[:user][:role] == 'custodian'
-        registers = Array.new
-
-        loop.with_index{|_, i|
-          register_name = resource_params[:teams_attributes][i.to_s][:registers]
+    binding.pry;
```

Comment:
I know you already said ignore this but this will give and exception in productions so im just reminding.